Source/Charts/Renderers/PieChartRenderer.swift

```diff
@@ -156,7 +160,7 @@ open class PieChartRenderer: DataRenderer
         element.accessibilityFrame = chart.bounds
         element.isHeader = true
         accessibleChartElements.append(element)
-
+        
```

Comment:
Make sure to not add whitespace on empty lines. There is an Xcode preference to prevent this under Text Editing